python/pyspark/pandas/tests/test_groupby.py

```diff
@@ -35,6 +35,19 @@
 
 
 class GroupByTest(PandasOnSparkTestCase, TestUtils):
+    def pdf(self):
```

Comment:
nit: let's add `@property` for `pdf` and `psdf` to:
1. Simply pdf init like `x=self.pdf` instead of `x=self.pdf()`
2. Keep the pdf/psdf property style with other tests
3. Make pdf/psdf read only, if some try to set pdf accidently will raise: `AttributeError: can't set attribute`

Reply:
That really makes sense. Thanks for explaining that!